source/MaterialXRuntime/Private/PvtMessage.cpp

```diff
@@ -52,6 +52,30 @@ RtCallbackId PvtMessageHandler::addSetAttributeCallback(RtSetAttributeCallbackFu
     return _callbackIdCounter++;
 }
 
+RtCallbackId PvtMessageHandler::addSetMetadataCallback(RtSetMetadataCallbackFunc callback, void* userData)
```

Comment:
How do you remove a callback?

Reply:
Looks like PvtSetMetadataCmd has a method:
`void removeCallback(RtCallbackId id);`
Which is used for that purpose, but I don't see it being used anywhere at the moment.

Reply:
`removeCallback` is on the message handler, the same class that has the various `addFooCalback` methods.